pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -106,7 +106,7 @@ func NewConfigObserver(
 			//apiserver.ObserveDefaultUserServingCertificate,
 			apiserver.ObserveNamedCertificates,
 			apiserver.ObserveUserClientCABundle,
-			apiserver.ObserveAdditionalCORSAllowedOrigins,
+			libgoapiserver.ObserveAdditionalCORSAllowedOrigins,
```

Comment:
@stlaz do you remember why we had a copy?

Reply:
it hasn't always been in library-go, I had a PR to move to library-go but I never pushed anyone hard enough to review it